bin/restore-db

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -eu
+CONFIRM_RESTORE=$1
+POSTGRES_SERVER_NAME=$2
+POSTGRES_USER=$3
+POSTGRES_DATABASE_NAME=$4
+POSTGRES_PASSWORD=$5
+BACKUP_FILENAME=$6
+
+if [[ -z "${CONFIRM_RESTORE}" ]]; then
+  echo "CONFIRM_RESTORE environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${POSTGRES_SERVER_NAME}" ]]; then
+  echo "POSTGRES_SERVER_NAME environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${POSTGRES_DATABASE_NAME}" ]]; then
+  echo "POSTGRES_DATABASE_NAME environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${POSTGRES_USER}" ]]; then
+  echo "POSTGRES_USER environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${POSTGRES_PASSWORD}" ]]; then
+  echo "POSTGRES_PASSWORD environment variable not set"
+  exit 1
+fi
+
+if [[ ! -f "${BACKUP_FILENAME}" ]]; then
+  echo "${BACKUP_FILENAME} does not exist."
+  exit 1
+else
+  export PGPASSWORD=${POSTGRES_PASSWORD}
```

Comment:
This may not have to be in the `if` block

Reply:
moved it out of `if` block.